packages/calcite-components/src/components/combobox/combobox.tsx

```diff
@@ -695,8 +702,31 @@ export class Combobox
     if (this.ignoreSelectedEventsFlag) {
       return;
     }
-
     const target = event.target as HTMLCalciteComboboxItemElement["el"];
+
+    const isSelectAllTarget = event
+      .composedPath()
+      .some((node) => (node as HTMLElement) === this.selectAllComboboxItemReferenceEl);
+
+    if (this.selectAllEnabled) {
+      if (isSelectAllTarget) {
+        this.toggleSelectAll();
+      }
+      this.updateAndGetSelectAllState();
+
+      // TODO: Doesn't hide the last element
+      if (this.isAllSelected()) {
+        this.selectedItems.forEach((item) => {
+          const chipEl = this.referenceEl.querySelector<Chip["el"]>(
+            `#${chipUidPrefix}${item.guid}`,
+          );
+          if (chipEl) {
+            this.hideChip(chipEl);
```

Comment:
Why are chips being hidden instead of just not being rendered?

Reply:
I'll start a discussion on why that was done before and create a refactoring issue with that in mind.